app/bundles/FormBundle/Controller/FormController.php

```diff
@@ -815,27 +836,35 @@ function ($a, $b) {
             $deletedActions = [];
         }
 
-        $leadFieldModel = $this->getModel('lead.field');
-        \assert($leadFieldModel instanceof FieldModel);
+//        $leadFieldModel = $this->getModel('lead.field');
+//        \assert($leadFieldModel instanceof FieldModel);
+//
+//        $mappedFields  = [];
+//        $mappedObjects = $entity->getMappedFieldObjects();
+//
+//        foreach ($mappedObjects as $mappedObject) {
+//            $mappedFields[$mappedObject] = $this->fieldCollector->getFields($mappedObject);
+//        }
```

Comment:
Wasn't this used for something?

Reply:
I think it is unnecessary code. `contactFields` and  `companyFields` are not used, so neither is `$leadFieldModel`. `mappedFields` and `mappedObjects` are not used anywhere either.

Reply:
@escopecz I think this code should be removed. WDYT?

Reply:
I removed the comented code.

Reply:
I retested this PR, and the removal of this code, did not impact the functionality of the code. so I think this conversation should be resolved and this PR be merged